scene/2d/animated_sprite_2d.cpp

```diff
@@ -154,32 +154,26 @@ void AnimatedSprite2D::_validate_property(PropertyInfo &p_property) const {
 void AnimatedSprite2D::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_INTERNAL_PROCESS: {
-			if (frames.is_null()) {
-				return;
-			}
-			if (!frames->has_animation(animation)) {
-				return;
-			}
-			if (frame < 0) {
```

Comment:
We generally don't frown upon expanded structures like the one that was here. It makes it clearer what conditions are checked and makes it easier to edit, even if it looks substantial and verbose.

Reply:
Would it be better as off as "||" and then newline for each condition? It's not the first time I've seen that in the codebase, but multiple guardrails *seemed* pretty rare to me. I can easily bring it back, anyway, regardless.

Reply:
> Would it be better as off as "||" and then newline for each condition?

I don't think so.

Reply:
But then there's lines like this that I didn't touch in the very same file. I thought it was more consistent go all in one line. Maybe it's a "rule" I don't quite understand.
https://github.com/godotengine/godot/blob/f9a04e10314781565442af93f4ba1757f5253180/scene/2d/animated_sprite_2d.cpp#L66
https://github.com/godotengine/godot/blob/f9a04e10314781565442af93f4ba1757f5253180/scene/2d/animated_sprite_2d.cpp#L82

Reply:
Well, to be clear, I didn't request any changes :) Just highlighted something that may not be required, since it doesn't affect the PR whatsoever.

Reply:
It's useful information for writing a cleaner codebase in the future, thank you

Reply:
I ended up leaving two early returns, in this PR, whenever applicable. One of them checks `frames` and the other checks `frame`.